dev/image/Dockerfile

```diff
@@ -71,7 +71,7 @@ ENV LEEWAY_NESTED_WORKSPACE=true
 RUN cd /usr/bin && curl -fsSL https://github.com/gitpod-io/leeway/releases/download/v0.2.7/leeway_0.2.7_Linux_x86_64.tar.gz | tar xz
 
 # dazzle
-RUN cd /usr/bin && curl -fsSL https://github.com/32leaves/dazzle/releases/download/v0.0.3/dazzle_0.0.3_Linux_x86_64.tar.gz | tar xz
+RUN cd /usr/bin && rm -f LICENSE README.md && curl -fsSL https://github.com/32leaves/dazzle/releases/download/v0.0.3/dazzle_0.0.3_Linux_x86_64.tar.gz | tar xz
```

Comment:
why `rm -f LICENSE README.md` ? :-D

Reply:
Whoops, we had some problems with docker builds in Leeway. That was me trying to fix it back when I didn't the problem was actually in leeway. I'll remove this change